docs/intro-HPC/2023/donphan-gallade.md

```diff
@@ -0,0 +1,92 @@
+# New Tier-2 clusters: `donphan` and `gallade`
+
+In April 2023, two new clusters were added to the HPC-UGent Tier-2 infrastructure: `donphan` and `gallade`.
+
+This page provides some important information regarding these clusters, and how they differ from the clusters
+they are replacing (`slaking` and `kirlia`, respectively).
+
+If you have any questions on using `donphan` or `gallade`, you can [contact the {{ hpcteam }}]({{ hpc_support_url }}).
+
+For software installation requests, please use the [request form](https://www.ugent.be/hpc/en/support/software-installation-request).
+
+---
+
+## `donphan`: debug/interactive cluster
+
+`donphan` is the new debug/interactive cluster.
+
+It replaces `slaking`, which will be retired on **Monday 22 May 2023**.
+
+It is primarily intended for interactive use: interactive shell sessions, using GUI applications through the
+[HPC-UGent web portal](../web_portal.md), etc.
+
+This cluster consists of 12 workernodes, each with:
+
+* 2x 18-core Intel Xeon Gold 6240 (Cascade Lake @ 2.6 GHz) processor;
+* one *shared* NVIDIA Ampere A2 GPU (16GB GPU memory)
+* ~738 GiB of RAM memory;
+* 1.6TB NVME local disk;
+* HDR-100 InfiniBand interconnect;
+* RHEL8 as operating system;
+
+To start using this cluster from a terminal session, first run:
+```
+module swap cluster/donphan
+```
+
+You can also start (interactive) sessions on `donphan` using the [HPC-UGent web portal](../web_portal.md).
+
+### User limits & overloading on `donphan`
+
+By imposing strict user limits and using oversubscription on this cluster,
+we ensure that anyone can quickly get a job with limited resources running, without having to wait in the queue.
```

Comment:
`quickly get a job with limited resources running, without having to wait in the queue` -> 
`get a job running without having to wait in the queue, albeit with limited resources` (yes, albeit ;)